apps/studio/components/interfaces/Account/Preferences/AccountDeletion.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+import Panel from 'components/ui/Panel'
+import { Alert } from 'ui'
+import DeleteAccountButton from './DeleteAccountButton'
+import { useProfile } from 'lib/profile'
+
+const AccountDeletion = () => {
```

Comment:
`export const AccountDeletion` - we're avoiding export defaults as a code guideline